### PR TITLE
update layout when font is changed

### PR DESCRIPTION
**Pull request by Igor-Palaguta**
Update layout

### CODE REVIEW

File: Example/JSBadgeView.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -103,7 +103,9 @@
 				6003F58B195388D20070C39A /* Products */,
 				A8BD1E7E9863D8176568336D /* Pods */,
 			);
+			indentWidth = 4;
```
Comment (JaviSoto) [inline Example/JSBadgeView.xcodeproj/project.pbxproj]: Do you mind not including this change? :)

**Comment by JaviSoto** [comment]
:shipit:, I'll merge just the change on the layout code, do you mind undoing the change on the project file?

**Comment by Igor-Palaguta** [comment]
I prefer to use 3 spaces for tab. So if some project uses another spaces count, we can set indentWidth just for one project. You can ignore this

**Comment by JaviSoto** [comment]
Oh, ok, I guess that's fine then :+1:

**Pull request merged**